VK/VK/Services/LocalService.swift

```diff
@@ -0,0 +1,52 @@
+// LocalService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import Realm
+import RealmSwift
+
+/// Сервис для локального хранения данных.
+struct LocalService {
+    // MARK: - Public Methods
+
+    func observeChanges<T: Object>(
+        type: T.Type,
+        notToken: inout RLMNotificationToken,
```

Comment:
```suggestion
        notToken: inout RLMNotificationToken,
```
Можно токен сохранить в сервисе. Ты к нему никогда не обращаешься.

Reply:
Встречал вроде что может исп несколько токенов. Посмотрим что сенсей скажет